src/main.py

```diff
@@ -208,7 +208,7 @@ async def add_process_time_header(request: Request, call_next):
 # Dependency
 v0_route_solar = "/v0/solar/GB"
 v0_route_system = "/v0/system/GB"
-
+app.include_router(national_router, prefix="/v0/solar/national")
```

Comment:
im not sure you need to add this, it should be in the route below already

Reply:
Yep, remove this as should work the same without 👍